src/infinite-list/index.ts

```diff
@@ -1,8 +1,23 @@
 import { consume, tag, Tag } from '@storefront/core';
 import List from '../list';
 
-@consume('infinite')
```

Comment:
Add `@provide('infiniteList', (props) => props)` so that yielded content can access this components props